VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-table.component.html

```diff
@@ -45,47 +45,47 @@
               <span *ngIf="hearing.panelMembers.length > 0">
                 <p class="govuk-!-margin-0 govuk-!-padding-0">
                   <span id="panel-member-count-{{ hearing.id }}">
-                    {{ hearing.panelMembers.length }} Panel Member{{ hearing.panelMembers.length > 1 ? 's' : '' }}
+                    {{ hearing.panelMembers.length }} {{ hearing.panelMembers.length > 1 ? 'Panel Members' : 'Panel Member' }}
                   </span>
                 </p>
               </span>
               <span *ngIf="hearing.wingers.length > 0">
                 <p class="govuk-!-margin-0 govuk-!-padding-0">
                   <span id="winger-count-{{ hearing.id }}">
-                    {{ hearing.wingers.length }} Winger{{ hearing.wingers.length > 1 ? 's' : '' }}
+                    {{ hearing.wingers.length }} {{ hearing.wingers.length > 1 ? 'Wingers' : 'Winger' }}
```

Comment:
translate?

Reply:
there are a couple like this in this file :)

Reply:
Fixed